about.html

```diff
@@ -0,0 +1,142 @@
+                20, Jan to 01 Feb</p>
+            <p class="loremtxt">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ut ullam exercitationem
+                deserunt corporis dignissimos sequi dolorum! Voluptatem, exercitationem facilis? Blanditiis sunt
+                accusantium quisquam maxime odio veniam mollitia doloribus eum. Aliquid laborum ipsum eum ullam
+                perferendis quidem modi tenetur dignissimos numquam? Cumque odio aut rerum impedit aliquam molestias
+                distinctio iure pariatur.</p>
+            <img src="assets/alogo/whlogo.png" alt="Web Hero code camp logo">
+        </section>
+
+        <section class="mentors past-camps">
+            <h2>See the past Web Hero Camps</h2>
+            <div class="line"></div>
+            <p>Take a look at the past Web Hero Fullstack Web Developer Code Camps which took place in 2021 and 2022
+                virtually.</p>
+            <div class="camps">
+                <div class="camp-card1"></div>
+                <div class="camp-card2"></div>
+            </div>
+
+        </section>
+
+        <section class="partner" id="partners">
+            <h2>Partners</h2>
+            <div class="line"></div>
+            <div class="container">
+                <div class="partner-all">
+                    <img src="assets/socials/mozilla.svg" alt="mozilla brand name">
+                    <img src="assets/socials/google.svg" alt="google brand name">
+                    <img src="assets/socials/naver.svg" alt="naver brand name">
+                    <img src="assets/socials/daum.svg" alt="daum brand name">
+                    <img src="assets/socials/airbnb.svg" alt="airbnb brand name">
+                </div>
+
+                <div class="partner1">
+                    <img src="assets/socials/mozilla.svg" alt="mozilla brand name">
+                    <img src="assets/socials/google.svg" alt="google brand name">
+                    <img src="assets/socials/naver.svg" alt="naver brand name">
+                </div>
+                <div class="partner2">
+                    <img src="assets/socials/daum.svg" alt="daum brand name">
+                    <img src="assets/socials/airbnb.svg" alt="airbnb brand name">
+                </div>
+            </div>
+        </section>
+        <section class="license">
+            <div class="container">
+                <img src="assets/alogo/whlogo.png" alt="license image">
+                <div class="license-info">
+                    <p>@ 2023 Creative Commons Bangladesh. Some Rights Reserved</p>
+                    <p class="korian-lan">ক্রিয়েটিভ কমন্স বাংলাদেশ। কিছু অধিকার সংরক্ষিত</p>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <script src="https://kit.fontawesome.com/19cf2cb812.js" crossorigin="anonymous"></script>
+    <script type="module" src="./scripts/script-about.js"></script>
+</body>
+
+</html>
```

Comment:
- [ ] good job so far addressing the previous reviewer suggestions 👏 , but it seems that you forgot to implement this comment from the previous reviewer , kindly fix all the issues mentioned in this comment 🙏 

![image](https://user-images.githubusercontent.com/89282221/235311745-4baaa1c5-ed5f-4817-aa5d-0211a0776ef4.png)

Reply:
Hi, @ ichala. 
Thanks for the review, however, I would like to mention that the above change is already made.
Thanks

Reply:
Please check the live deployment hrere : https://porag-m06.github.io/Porag-s-Web-Hero-23/index.html

Reply:
@porag-m06 im not only using the same link that you sent me but also the local version , you can check in here https://porag-m06.github.io/Porag-s-Web-Hero-23/about.html that the footer is still white and the partner section is still not removed

Reply:
![image](https://user-images.githubusercontent.com/89282221/235312982-8faf0743-d8cf-4d38-b11a-cb13eeeef9d3.png)

Reply:
@ichala I got confused in a hurry, My appologis. Thank for the replay